modules/tinymce/src/core/test/ts/browser/html/StylesTest.ts

```diff
@@ -33,18 +33,25 @@ describe('browser.tinymce.core.html.StylesTest', () => {
   it('Colors force hex and lowercase', () => {
     const styles = Styles();
 
-    assertStyles(styles, 'color: rgb(1,2,3)', 'color: rgb(1,2,3);');
-    assertStyles(styles, 'color: RGB(1,2,3)', 'color: rgb(1,2,3);');
+    assertStyles(styles, 'color: rgb(1,2,3)', 'color: #010203;');
+    assertStyles(styles, 'color: RGB(1,2,3)', 'color: #010203;');
     assertStyles(styles, 'color: #FF0000', 'color: #ff0000;');
-    assertStyles(styles, '  color:   RGB  (  1  ,  2  ,  3  )  ', 'color: rgb  (  1  ,  2  ,  3  );');
+    assertStyles(styles, '  color:   RGB  (  1  ,  2  ,  3  )  ', 'color: #010203;');
     assertStyles(styles,
       '   FONT-SIZE  :  10px  ;   COLOR  :  RGB  (  1  ,  2  ,  3  )   ',
-      'font-size: 10px; color: rgb  (  1  ,  2  ,  3  );'
+      'font-size: 10px; color: #010203;'
     );
     assertStyles(styles,
       '   FONT-SIZE  :  10px  ;   COLOR  :  RED   ',
       'font-size: 10px; color: red;'
     );
+    assertStyles(
+      styles,
+      'border: 1px solid rgb(255, 0, 0);',
+      'border: 1px solid rgb(255, 0, 0);'
+      // TODO: color in border style should be in HEX format once https://ephocks.atlassian.net/browse/TINY-9817 is fixed.
```

Comment:
Ticket number doesn't seem to be correct 🤔

Reply:
Whoops, typo